rustfmt.toml

```diff
@@ -1,5 +1,5 @@
 comment_width = 100
 edition = "2021"
 group_imports = "One"
-imports_granularity = "Module"
```

Comment:
a change like this is worth doing in a separate PR as it changes a lot of the files without touching the functionality. This makes tracking the actual logic changes a lot harder